src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -65,9 +67,75 @@ public AgentSettings LoadSettings()
             return settings;
         }
 
+        public void checkAgentRootDirectorySecure()
```

Comment:
Should this method be private?

Reply:
Thanks for the point, it makes sense, I will fix it.

Reply:
Fixed it.